modules/burdastyle_thunder_admin/burdastyle_thunder_admin.module

```diff
@@ -236,3 +236,27 @@ EOT;
 
   return $cssPlaceholder;
 }
+
+
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter().
+ */
+function burdastyle_thunder_admin_field_widget_paragraphs_form_alter(&$element, &$form_state, $context) {
+  if ($element['#paragraph_type'] == 'content_reference') {
+    foreach ($element['subform']['field_e_view_mode']['widget']['#options'] as $key => $value) {
```

Comment:
check if $element['subform']['field_e_view_mode']['widget']['#options'] is set

Reply:
done